PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Commands/AssertPSCompatibilityProfileIsValidCommand.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerShell.CrossCompatibility.Collection;
+using Microsoft.PowerShell.CrossCompatibility.Data;
+using Microsoft.PowerShell.CrossCompatibility.Utility;
+
+namespace Microsoft.PowerShell.CrossCompatibility.Commands
+{
+    /// <summary>
+    /// Class defining the Assert-PSCompatibilityProfileIsValid cmdlet.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Assert, CommandUtilities.MODULE_PREFIX + "ProfileIsValid")]
```

Comment:
it's also possible this could be the verb `Test` and return `true` or `false`

Reply:
Yeah I was weighing up those two. The problem is always that a boolean gives no diagnosis. I don't like that it throws an exception really, but it's better than a third custom solution